.changeset/long-keys-watch.md

```diff
@@ -0,0 +1,6 @@
+---
+'firebase': minor
+'@firebase/vertexai': minor
```

Comment:
I guess you will have to change this after https://github.com/firebase/firebase-js-sdk/pull/9011 is merged.

Reply:
good point, I'll hold off on merging this until the packages have settled.

Reply:
done